opm/material/fluidsystems/BlackOilFluidSystem.hpp

```diff
@@ -963,28 +1030,13 @@ class BlackOilFluidSystem : public BaseFluidSystem<Scalar, BlackOilFluidSystem<S
         assert(0 <= regionIdx && regionIdx <= numRegions());
 
         const auto& p = Opm::decay<LhsEval>(fluidState.pressure(phaseIdx));
-        const auto& T = Opm::decay<LhsEval>(fluidState.temperature(phaseIdx));
-
-        switch (phaseIdx) {
-        case oilPhaseIdx:
-            return
-                oilPvt_->internalEnergy(regionIdx, T, p, Opm::BlackOil::template getRs_<ThisType, FluidState, LhsEval>(fluidState, regionIdx))
-                + p/density<FluidState, LhsEval>(fluidState, phaseIdx, regionIdx);
-
-        case gasPhaseIdx:
-            return
-                gasPvt_->internalEnergy(regionIdx, T, p, Opm::BlackOil::template getRv_<ThisType, FluidState, LhsEval>(fluidState, regionIdx))
-                + p/density<FluidState, LhsEval>(fluidState, phaseIdx, regionIdx);
+        //const auto& T = Opm::decay<LhsEval>(fluidState.temperature(phaseIdx));
+        //double ent_fac=0;
+        double ent_fac = 1.0;
```

Comment:
camelCase please.

Reply:
I think for consistency with E100 and some simple investigation it had been use full to have this entfac as parameter. but it need to be consistent between the fluid state and the fluid system in the way. (If entFace = 0, enthalpy effect is ignored and one should temprature should stay within boundary conditions if I am not wrong...)

Reply:
is this an ECLIPSE parameter, i.e. can it somehow be set from the deck?